File: three-consecutive-odds.ts
{
    const helper = (num: number) => {
        return num % 2 !== 0 
    }   

    function threeConsecutiveOdds(arr: number[]): boolean {
        
        for (let i = 2; i < arr.length; i++) {

            if (helper(arr[i]) && helper(arr[i - 1]) && helper(arr[i - 2]))
                return true

        }

        return false
    };

    console.log(threeConsecutiveOdds([1,2,34,3,4,5,7,23,12]))
}

File: three-consecutive-odds.rs
pub fn three_consecutive_odds(arr: Vec<i32>) -> bool {
    let mut c = 0;

    for item in arr {
        if item % 2 != 0 {
            c += 1;

            if c == 3 {
                return true;
            }
        } else {
            c = 0;
        }
    }

    return false;
}
